config/locales/fr.yml

```diff
@@ -1,21 +1,21 @@
 fr:
```

Comment:
I think you can delete this file.

Since we are not french users and the traductions here are mostly done by google translate, I think we can drop it.

For our internal plugin we already drop it.

You can ask for confirmation as well.

Reply:
L-am sters